docs/test_feature_units_in_docs.py

```diff
@@ -0,0 +1,38 @@
+"""Parse eFeatures.rst.
+    Make sure every unit there is present in efel.units
+"""
+
+from pathlib import Path
+import re
+
+
+from efel import units
+
+
+def extract_feature_and_units(rst_text: str) -> dict:
+    """Parse rst to extract features and units."""
+    # Use regex to match and extract the desired information
+    pattern = r'LibV\d+\s*:\s*([\w_]+)[\s\S]*?- \*\*Units\*\*:\s*([\w/\(\)]+)'
```

Comment:
I will just trust you with that one ^_^'

Reply:
Holy wow, that's something. 

I ask GPT4 for a translation:
```

ME: What does this regex line do ?
pattern = r'LibV\d+\s*:\s*([\w_]+)[\s\S]*?- \*\*Units\*\*:\s*([\w/\(\)]+)'

GPT4: This regular expression pattern is designed to match and capture specific information from a string. Let's break down the pattern to understand it better:

LibV: This part matches the literal string "LibV".
\d+: This part matches one or more digits (0-9).
\s*: This part matches zero or more whitespace characters (spaces, tabs, newlines, etc.).
:: This part matches the literal colon character.
\s*: This part (again) matches zero or more whitespace characters.
([\w_]+): This part matches one or more word characters (letters, digits, or underscores) and captures them in a group.
[\s\S]*?: This part matches zero or more occurrences of any character, including whitespace and non-whitespace characters, in a non-greedy fashion (i.e., as few characters as possible).
- \*\*Units\*\*:: This part matches the literal string "- Units:".
\s*: This part (again) matches zero or more whitespace characters.
([\w/\(\)]+): This part matches one or more word characters (letters, digits, or underscores), forward slashes, or parentheses, and captures them in a group.
In summary, this regex pattern searches for a string that starts with "LibV" followed by one or more digits, a colon, and one or more word characters or underscores (capturing this part). Then, it matches any number of characters until it finds the string "- Units:" followed by one or more word characters, forward slashes, or parentheses (capturing this part as well).

The pattern is typically used with Python's re module to match and extract information from a larger string. The first captured group would represent a name or identifier, and the second captured group would represent some unit information.

```

Reply:
Done in the next commit :)